.github/dependabot.yml

```diff
@@ -1,4 +1,6 @@
+---
```

Comment:
this isn't how YAML should start

Reply:
In fact it is, see section 2.2 of the YAML specification: https://yaml.org/spec/1.2.2/

Reply:
It's only mandatory to have them if you use directives in the YAML document. Directives are optional for usage, please see [section 3.2.3.4](https://yaml.org/spec/1.2.2/#3234-directives) of the YAML specification. Please see examples in section [6.8.1](https://yaml.org/spec/1.2.2/#681-yaml-directives) to have a better understanding.

Reply:
Indeed its not mandatory and we can choose to leave it out. But stating "this isn't how YAML should start" isn't accurate.